cmd/managementd/main.go

```diff
@@ -47,6 +47,9 @@ func main() {
 		return
 	}
 	log.Printf("config: %v", config)
+	if config.Port != 80 {
+		log.Printf("warning: avahi service is advertised on port 80 but port %v is being used", config.Port)
```

Comment:
Good thinking to log this.

The other way to handle this is to have the management interface register the service with avahi dynamically (using its D-bus API) but that's more complicated and probably not necessary. We're only likely to want to change the port during development.